lib/ebpd_utils.h

```diff
@@ -0,0 +1,30 @@
+#ifndef EBPD_UTILS_H_
```

Comment:
The style of include guards seems to be:
#define LIB_EBPD_UTILS_H_ if we follow the pattern of lib/ebpd.h